setup.py

```diff
@@ -27,20 +27,19 @@ def find_version(*file_paths):
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["Django>=2.2", "requests>=2.21.0"],
+    install_requires=["Django>=3.2", "requests>=2.26.0"],
```

Comment:
TL;DR the rest is mostly metadata, so this is really the only impact here — basically moving the requires in line with what is pinned for testing, to have the visibility into confirming it truly working. Feel free to revert if you feel like still keeping the older (and fully working as of now) versions around.